includes/webhook.php

```diff
@@ -307,6 +309,18 @@ function format_shortcodes($content) {
     return $content;
 }
 
+/**
+ * Replaces html entities that are used in gutenberg blocks.
+ */
+function decode_gutenberg_blocks_html_entities($content) {
+
```

Comment:
You're replacing **all** `&quot;` with `'`
What you want to do is
1. grab the piece of content between `&lt;!--` and `--&gt;`
2. do the replaces only on that piece of content
3. replace that part inside the full content.